db/migrate/20181207162311_create_legal_aid_application_restrictions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLegalAidApplicationRestrictions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :legal_aid_application_restrictions, id: :uuid do |t|
+      t.references(
+        :legal_aid_application,
+        foreign_key: true,
+        type: :uuid,
+        index: { name: 'laa_id_laa_restriction_id' }
```

Comment:
the default index name is too long?

Reply:
Exactly. The migration wouldn't migrate until I set a shorter name.